src/scenarios/shared/runner.py

```diff
@@ -60,9 +60,10 @@ def parseargs(self):
         parser = ArgumentParser()
         subparsers = parser.add_subparsers(title='subcommands for sdk tests', required=True, dest='testtype')
         startupparser = subparsers.add_parser(const.STARTUP)
+        startupparser.add_argument('--scenario-name', dest='scenarioname')
```

Comment:
Can this go on the root parser? Does each sub-parser always have to include this?

Reply:
It could but this would make it mandatory to position the optional argument before the required sub-parsers. For example, ```python test.py ---scenario-name 'empty console template' startup``` is a bit counter-intuitive.

Reply:
Fair. Could we build a little factory or something? We will have more parsers in the future I suspect.